spec/jobs/regular/publish_group_membership_updates_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
```suggestion
```

This is no longer required since it is configured in rspec: https://github.com/discourse/discourse/blob/83f4e4566425acf544b5f686f89a977e7d94a005/.rspec#L2